extension/background/languages.js

```diff
@@ -0,0 +1,32 @@
+this.languages = {};
```

Comment:
Looks like this variable isn't needed anywhere

Reply:
Fixed.